amivapi/utils.py

```diff
@@ -90,17 +90,28 @@ def mail(to, subject, text):
     subject = app.config['API_MAIL_SUBJECT'].format(subject=subject)
 
     if app.config.get('TESTING', False):
-        app.test_mails.append({
+        if replyTo is not None:
+            app.test_mails.append({
+                'subject': subject,
+                'from': sender,
+                'receivers': to,
+                'text': text,
+                'reply-to': replyTo
+            })
+        else: 
+          app.test_mails.append({
             'subject': subject,
             'from': sender,
             'receivers': to,
             'text': text
-        })
+        })  
     elif config.SMTP_SERVER and config.SMTP_PORT:
         msg = MIMEText(text)
         msg['Subject'] = subject
         msg['From'] = sender
         msg['To'] = ';'.join([to] if isinstance(to, str) else to)
+        if replyTo is not None:
+            msg['reply-to'] = replyTo
```

Comment:
```suggestion
            msg['Reply-To'] = replyTo
```

Email headers are generally written with captital letters.